locale/en/docs/es6.md

```diff
@@ -10,7 +10,7 @@ All ECMAScript 2015 (ES6) features are split into three groups for **shipping**,
 
 * All **shipping** features, which V8 considers stable, are turned **on by default on Node.js** and do **NOT** require any kind of runtime flag.
 * **Staged** features, which are almost-completed features that are not considered stable by the V8 team, require a runtime flag: `--es_staging` (or its synonym, `--harmony`).
```

Comment:
I think there are valid reasons why features might be toggled individually - it's also part of the Node API, so I reckon we should document it unless it's actually removed from the API